schemas/validation_tests/individual-valid-1.json

```diff
@@ -0,0 +1,41 @@
+{
+    "object_id": {
+        "alias" : "NM305004", 
+        "center_name" : "EBI-TEST",
+        "ega_accession": "EGAI00000000001"
+    },
+    "minimal_public_attributes": {
+        "subject_id": "donor-NM305004",
+        "biological_sex": "male",
+        "phenotype": {
+            "experimental_condition_term": "fibroadenoma",
+            "experimental_condition_curie": "EFO:1000254"
+        }
+    },
```

Comment:
I really like how this clearly organises the pubic attributes in a way that should be easily understood by both submitters and requesters!